sys/freebsd/posix_fs.txt

```diff
@@ -38,11 +41,16 @@ freebsd11_stat(file ptr[in, filename], statbuf ptr[out, freebsd11_stat])
 freebsd11_fstat(fd fd, statbuf ptr[out, freebsd11_stat])
 freebsd11_lstat(file ptr[in, filename], statbuf ptr[out, freebsd11_stat])
 fstatat(fd fd, path ptr[in, filename], statbuf ptr[out, stat], flag flags[fstatat_flags])
+freebsd11_fstatat(fd fd, path ptr[in, filename], statbuf ptr[out, freebsd11_stat], flag flags[fstatat_flags])
+freebsd11_getfsstat(buf ptr[out, freebsd11_statfs], size intptr, mode flags[getfsstat_flags])
+getfsstat(buf ptr[out, statfs], size intptr, mode flags[getfsstat_flags])
```

Comment:
Shouldn't the second parameter be the length of `buf`?

Reply:
I thought that, but then I looked up similar getfsstat call in NetBSD. It used intptr for long ig. I will change it to len[buf].